static/js/faculty_carousel.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Slider from 'react-slick';
+
+
+class FacultyTile extends React.Component {
+  props: {
+    name:     string,
+    title:    string,
+    imageUrl: string,
+    bio:      string,
+  }
+  render() {
+    let nameStr;
+    if (this.props.title) {
```

Comment:
Can you use destructuring here: `const { name, title, bio } = this.props;` for example